tests/fsharpqa/Source/CompilerOptions/fsi/langversion/comparer.fsx

```diff
@@ -0,0 +1,32 @@
+// #NoMT #CompilerOptions #RequiresENU   
```

Comment:
Can you just use one of the existing `comparer.fsx` already checked in to `fsharpqa` - I've improved this comparer from time to  time but we keep duplicating it

Reply:
@dsyme, I think a separate PR unifying the comparers if possible is the way.